config/auth.go

```diff
@@ -0,0 +1,17 @@
+package config
+
+import (
+	"os"
+
+	"github.com/streampets/backend/repositories"
+	"github.com/streampets/backend/services"
+	"gorm.io/gorm"
+)
+
+func CreateAuthService(db *gorm.DB) services.AuthService {
+	clientSecret := os.Getenv("CLIENT_SECRET")
```

Comment:
you may want to bullet-proof these getenv calls.  If this is going to third parties, it's very possible that your users will miss one or more of these.  I'd add some error handling and some validation to all the environment variables you are pulling. in.